tests/docs/smoke-all/2024/01/18/8331.qmd

```diff
@@ -0,0 +1,17 @@
+---
+title: foo
+format: docusaurus-md
+_quarto:
+  tests:
+    docusaurus-md:
+      ensureFileRegexMatches:
+        - ["export const quartoRawHtml"]
+        - []
+---
+
+```{python}
+#| filename: Python
+import polars
+
+polars.DataFrame({"foo": [1, 2,]})
+```
```

Comment:
Suggestion to avoid using Python chunk here as this adds some dependencies and requires Jupyter process. 

I don't know if `polars` is in our current python dependency by the way. We could add for sure, but the example shared in https://github.com/quarto-dev/quarto-cli/issues/8331#issuecomment-1898319691 triggered the bug too 

````markdown
---
title: foo
format: docusaurus-md
---

```{=html}
<span class="dummy">Content</span>
```
````

Unless you want to specifically test with filename attribute and with python cells. In that case ignore my comment

Reply:
Sure, that's a good idea.

Reply:
Done.